/var/spool/slurmd/job12584826/slurm_script: line 20: /scratch/user/k1411130/NEHD_NLBP: Is a directory
/scratch/user/k1411130/NEHD_NLBP/venv/lib/python3.11/site-packages/medmnist/info.py:17: UserWarning: Failed to setup default root.
  warnings.warn("Failed to setup default root.")
Using device: cuda
Starting Batch Experiments...
Implementing PRMI
Implementing PRMI as Conv Fusion or Indepedently
Traceback (most recent call last):
  File "/scratch/user/k1411130/NEHD_NLBP/demo.py", line 350, in <module>
    main(args,params)
  File "/scratch/user/k1411130/NEHD_NLBP/demo.py", line 65, in main
    dataloaders_dict = Prepare_DataLoaders(data_parameters)
                       ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/user/k1411130/NEHD_NLBP/Prepare_Data.py", line 181, in Prepare_DataLoaders
    train_dataset = PRMIDataset(data_dir,subset='train',transform=initial_transform['train'])
                    ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/scratch/user/k1411130/NEHD_NLBP/Datasets/PRMIDataset.py", line 38, in __init__
    with open(lab_json) as f:
         ^^^^^^^^^^^^^^
FileNotFoundError: [Errno 2] No such file or directory: './Datasets/PRMI/train/labels_image_gt/Cotton_736x552_DPI150_train.json'
/scratch/user/k1411130/NEHD_NLBP/venv/lib/python3.11/site-packages/medmnist/info.py:17: UserWarning: Failed to setup default root.
  warnings.warn("Failed to setup default root.")
Generating Batch Experiments Results...
Saved_Models/Classification//DSA/Fashion_MNIST/Run_1
Done!
Confusion matrix, without normalization
**********Run 1 Finished**********
Saved_Models/Classification//DSA/Fashion_MNIST/Run_2
Done!
Confusion matrix, without normalization
**********Run 2 Finished**********
Saved_Models/Classification//DSA/Fashion_MNIST/Run_3
Done!
Confusion matrix, without normalization
**********Run 3 Finished**********
Confusion matrix, without normalization
Finished setting 1 of 16
